module/zfs/zfs_sa.c

```diff
@@ -69,6 +71,8 @@ sa_attr_reg_t zfs_attr_table[ZPL_END+1] = {
 	{NULL, 0, 0, 0}
 };
 
+int zfs_zil_saxattr_enabled = 1;
```

Comment:
No reason this can't be `static` I think?

Reply:
Done